components/SupportOppose/SupportOppose.tsx

```diff
@@ -0,0 +1,116 @@
+import { useMemo, useCallback, useState, CSSProperties } from "react";
+import { FaCheck, FaTimes } from "react-icons/fa";
+import { getRandomInt, getRandomBool } from "utils/numbers";
+import { addAlphaToHexColor } from "utils/strings";
+import useDocumentBaseStyle from "hooks/useDocumentBaseStyle";
+import styles from "./SupportOppose.module.scss";
+
+/** This section will pull from generated.ts when we have data */
+
+type SupportOpposeType = "support" | "oppose";
```

Comment:
There is currently an `ArgumentPosition` type that you can use here from generated.